Application.mk

```diff
@@ -189,8 +192,23 @@ install:: $(PROGLIST)
 
 else
 
-MAINNAME := $(addsuffix _main,$(PROGNAME))
+MAINNAME := $(subst ",,$(addsuffix _main,$(PROGNAME)))
```

Comment:
Do we still need change this line after https://github.com/apache/incubator-nuttx-apps/pull/1407/files#r1021549899

Reply:
quite right sir!